fiftyone/utils/openworld_sam.py

```
@@ -0,0 +1,397 @@
+            detections_per_image=config.top_k,
+        )
+        self._hf_model = OpenWorldSAMModel.from_pretrained(
+            local_dir,
+            config=hf_config,
+            low_cpu_mem_usage=False,
+        )
+        self._hf_model.eval()
+        return self._hf_model
+
+    @staticmethod
+    def _to_uint8(img):
```

Ruff (0.15.18)

[warning] Missing return type annotation for staticmethod `_to_uint8`

(ANN205)